PrimeNumbers.java

```diff
@@ -15,12 +15,41 @@ public static void myassert(boolean  x) {
     }
 
     private static int secondMaxSixDigitPrime(){
-        //WRITE YOUR CODE HERE
+        int num,i,check,max=0;
+        for (check = 999999; check >= 100000; check--){
```

Comment:
**Better Readability** - declare MIN =100000 and MAX=999999 and use them in the for loop